base/workerpool.jl

```diff
@@ -0,0 +1,75 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+
+type WorkerPool
+    channel::RemoteChannel{Channel{Int}}
```

Comment:
Need to add a default constructor here to avoid the following build warning : 
`WARNING: Method definition (::Type{Base.WorkerPool})(Any) in module Base at workerpool.jl:5 overwritten at workerpool.jl:17`